web/openapi/openapi.yaml

```diff
@@ -19,7 +19,7 @@ paths:
   /export:
     get:
       summary: Export data
-      description: Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/language)
+      description: Export data from VAST according to a query. The query must be a valid expression in the VAST Language. (see https://vast.io/docs/understand/language)
```

Comment:
```suggestion
      description: Export data from VAST according to a query. The query must be a valid expression in the VAST language. (see https://vast.io/docs/understand/language)
```

Reply:
There are a few more occurrences of this kind in this PR that I didn't flag.

Reply:
Replaced "VAST Language" with "VAST language" in https://github.com/tenzir/vast/pull/2959/commits/4b289bb23fce3c786bad9b58305f8910e4d0182a